std/assembly/builtins.ts

```diff
@@ -925,6 +937,18 @@ export namespace f64 {
   @lazy
   export const MAX_SAFE_INTEGER: f64 =  9007199254740991;
 
+  // @ts-ignore: decorator
+  @lazy
+  export const POSITIVE_INFINITY: f64 =  Infinity;
+
+  // @ts-ignore: decorator
+  @lazy
+  export const NEGATIVE_INFINITY: f64 = -Infinity;
+
+  // @ts-ignore: decorator
+  @inline @lazy
+  export const NaN: f64 = 0.0 / 0.0;
```

Comment:
can't use `export const NaN: f64 = NaN` here, so I decide use this workaround

Reply:
Another workaround might be to import `NaN` from the same file under a different name, like `import { NaN as builtin_NaN } from "./number";` and then do `export const NaN: f64 = builtin_NaN;`. Not sure what's less strange.

Reply:
I guess `import { NaN as builtin_NaN } from ..` more strange and expensive in term of compilation cost